go/exocom/src/security/manager.go

```diff
@@ -33,6 +37,17 @@ func (m *Manager) ReceiveMessage(message structs.Message) *Result {
 	var messageToSend *structs.Message
 	warningMessage := ""
 	switch message.Name {
+	case "security request":
+		bytes, err := json.Marshal(message.Payload)
+		if err != nil {
+			warningMessage = errors.Wrap(err, "Marshaling 'security request' payload").Error()
```

Comment:
I believe we need a `break` after this to jump to the end of the switch statement and prevent processing any further. Same for the other instance we set warningMessage